client/components/data-labels/project-version-label.template.html.slim

```diff
@@ -1,6 +1,6 @@
-span.data-label
+span.data-label tooltip='{{ getTooltip() }}' tooltip-enable='tooltipEnabled()'
   '
   span.label.label-primary ng-if='!versionOnly'
-    a.label-link ng-if='linkable' href='#' ui-sref='org.projects.show({ orgName: organization.name, projectName: project.name })' {{ project | projectName }} {{ projectVersion }}
-    span ng-if='!linkable' {{ project | projectName }} {{ projectVersion }}
-  span.label.label-primary ng-if='versionOnly'  {{ projectVersion }}
+    a.label-link ng-if='linkable' href='#' ui-sref='org.projects.show({ orgName: organization.name, projectName: project.name })' {{ getLabel() }}
+    span ng-if='!linkable' {{ getLabel() }}
+  span.label.label-primary ng-if='versionOnly' {{ getLabel() }}
```

Comment:
You're assuming that we'll never want to display version-only labels with a link. I would make no such assumption, especially as it makes the directive arguments counter-intuitive.